__tests__/app.test.js

```diff
@@ -215,6 +215,80 @@ describe("GET /api/articles", () => {
         );
       });
   });
+  test("status 200 return article array sorted by author ", () => {
```

Comment:
This description should be more generic please! It's not specifically testing for sorted_by author, it's just that there's a sort_by in general. We're just using author in this case to prove that it works